src/App.tsx

```diff
@@ -25,37 +31,91 @@ const useStyles = makeStyles((theme) => ({
   selectEmpty: {
     marginTop: theme.spacing(2),
   },
+  color: {
```

Comment:
this isn't a good name because color can mean many things. Usually we pick either a name that indicates what component it is applied to, or what it does. So I suggest you call it whiteBgColor.